doc/tasks.rst

```diff
@@ -168,6 +168,25 @@ check :ref:`tools.LongRunning<tools.LongRunning>` as an example.
 
 
 
+action arguments
+-----------------
+
+You always can pass some arguments into your action in case it's a `python-action`.
+As shown above, it's expected if action is python callable or tuple of `(callable, *args, **kwargs)` kind.
+In both cases only `callable` is required, ``args`` and ``kwargs`` may be skipped or empty.
+
+For ``args`` may be accepted tuple or list object.
+
+.. literalinclude:: tutorial/task_args.py
+
+For ``kwargs`` only `dict` object may be accepted.
+
+.. literalinclude:: tutorial/task_kwargs.py
+
+Also you may be interested in :ref:`params<task_args.arguments>` and :ref:`getargs<dependencies.getargs>` for passing arguments into actions.
```

Comment:
I try to avoid reference concepts that were not explained yet because first time readers wont be able to understand this.